documentation/guide-upgrade-devon4ng.asciidoc

```diff
@@ -31,19 +31,19 @@ In order to perform a basic upgrade we will execute:
 $ ng update @angular/cli @angular/core
 ```
 
-=== Upgrade from Angular 8 to Angular 9
+=== Upgrade from Angular 9 to Angular 10
```

Comment:
Maybe now we can already generalize this to avoid version numbers unless we have to do something specific in a concrete version:

```suggestion
=== Upgrade to new Angular version
```